app/src/main/res/layout/activity_bill.xml

```diff
@@ -0,0 +1,287 @@
+
+                                <com.toptoche.searchablespinnerlibrary.SearchableSpinner
+                                    android:layout_width="match_parent"
+                                    android:layout_height="wrap_content"
+                                    android:scrollbarSize="@dimen/dp_20"
+                                    bind:hintText="@string/title_stylist"
+                                    bind:searchableSpinnerAdapter="@{viewModel.stylistAdapter}"
+                                    bind:stylist="@={viewModel.stylist}"/>
+                            </LinearLayout>
+                        </android.support.v7.widget.CardView>
+
+                        <android.support.design.widget.TextInputLayout
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:paddingEnd="@dimen/dp_8"
+                            android:paddingStart="@dimen/dp_8">
+
+                            <EditText
+                                android:layout_width="match_parent"
+                                android:layout_height="wrap_content"
+                                android:hint="@string/title_price"
+                                android:inputType="number"
+                                android:text="@={viewModel.price}"
+                                android:textSize="@dimen/sp_10"/>
+                        </android.support.design.widget.TextInputLayout>
+
+                        <android.support.design.widget.TextInputLayout
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:paddingEnd="@dimen/dp_8"
+                            android:paddingStart="@dimen/dp_8">
+
+                            <EditText
+                                android:layout_width="match_parent"
+                                android:layout_height="wrap_content"
+                                android:hint="@string/title_number"
+                                android:inputType="number"
+                                android:text="@={viewModel.qty}"
+                                android:textSize="@dimen/sp_10"/>
+                        </android.support.design.widget.TextInputLayout>
+                    </LinearLayout>
+
+                    <LinearLayout
+                        android:layout_width="wrap_content"
+                        android:layout_height="match_parent"
+                        android:gravity="center"
+                        android:orientation="vertical"
+                        android:padding="@dimen/dp_10">
+
+                        <TextView
+                            android:layout_width="wrap_content"
+                            android:layout_height="wrap_content"
+                            android:layout_marginStart="@dimen/dp_5"
```

Comment:
ERROR: [RtlCompat] To support older versions than API 17 (project specifies 15) you should *also* add `android:layout_marginLeft="@dimen/dp_5"`
 API 17 adds a `textAlignment` attribute to specify text alignment. However, if you are supporting older versions than API 17, you must *also* specify a gravity or layout_gravity attribute, since older platforms will ignore the `textAlignment` attribute.